Adding Tls client information (#1344)
temporalio/documentation: docs/python/how-to-set-the-namespace-for-a-temporal-client-in-python.md

To specify a Namespace, set the `namespace` parameter from the [`connect()`](https://python.temporal.io/temporalio.client.client#connect) method.

```python
await Client.connect("127.0.0.1:7233", namespace="your-custom-namespace")
```

> `Client` may be directly instantiated with a service of another. For example, if you need to create another Client to use an additional Namespace.

Reviewer: ```suggestion
`Client` can be directly instantiated with a service of another, such as when you need to create another Client to use an additional Namespace.
```
Or similar (to avoid incomplete sentence [like this one]).
